README.md

```diff
@@ -0,0 +1,108 @@
+########################################################
+#			Welcome to the Kokkos Documentation Workflow     #
+########################################################
+
+	Here, we summarize the steps for setting up the kokkos-core-wiki repo, and a
+branch-based workflow for updating documentation in the main Kokkos project.  The kokkos-core-wiki is the
+canonical source of Kokkos documentation "Truth".
+
+	CAVEAT: Creation of wholly new pages will require Review and Approval in some
+cases.  Check with your team lead, if you are uncertain.
```

Comment:
This repository is also meant for external (non-Kokkos, non-Sandia) contributors for whom this paragraph sounds confusing/not encouraging to participate.